### PR TITLE
[nameof] Update to 0.9.2

### PR DESCRIPTION
**Pull request by Neargye**
https://github.com/Neargye/nameof/releases/tag/v0.9.2

### CODE REVIEW

File: ports/nameof/portfile.cmake [hunk 1]
```diff
@@ -20,12 +20,12 @@ vcpkg_configure_cmake(
 
 vcpkg_install_cmake()
 
-vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/${PORT})
```
Comment (ehsan-mohammadi) [inline ports/nameof/portfile.cmake]: @Neargye Can you tell me why you change `${PORT}` to `nameof`?

Comment (Neargye) [reply ports/nameof/portfile.cmake]: Several of my packages are in vcpkg.
I can’t say that such a record is better, but it’s more convenient purely for me.

Example:
* Lib = magic_enum
* ${PORT} =  magic-enum
    *  vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/${PORT}) // Error
    * vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/magic_enum) // OK

So not using this ${PORT} just helps me personally to navigate the record.

Comment (ehsan-mohammadi) [reply ports/nameof/portfile.cmake]: As you said, for some ports like `magic_enum` it's a good idea. BTW thanks for letting me know.

Comment (Neargye) [reply ports/nameof/portfile.cmake]: In this case, I just left the same template from the last project.

**Pull request merged**